core/src/Ribasim.jl

```diff
@@ -46,8 +47,7 @@ using Graphs:
     rem_edge!
 
 using Legolas: Legolas, @schema, @version, validate, SchemaVersion, declared
-using Logging: current_logger, min_enabled_level, with_logger
-using LoggingExtras: EarlyFilteredLogger, LevelOverrideLogger
+using Logging: with_logger, LogLevel, AbstractLogger, SimpleLogger
```

Comment:
```suggestion
using Logging: with_logger, LogLevel, AbstractLogger
```

SimpleLogger isn't used.